_utils/gitlabci.sh

```diff
@@ -48,6 +48,9 @@ git submodule update --init --recursive
 
 echo -e "section_end:$(date +%s):prepare\r\e[0K"
 
+#temp fixup
+git remote set-url origin $(git remote get-url origin | sed -e s/gitlab/github/)
+
```

Comment:
I can't say whether this is right or wrong. I'm not an expert here. To be reviewed by someone else.